src/PlugWallet/index.ts

```diff
@@ -213,7 +214,7 @@ class PlugWallet {
     return balance;
   };
 
-  public getNFTInfo = async ({ canisterId, standard }) => {
+  public getNFTInfo = async ({ canisterId, standard }): Promise<WalletNFTInfo> => {
     const nft = await this.network.getNftInfo({ canisterId, identity: this.identity, standard });
     return nft;
```

Comment:
Could we return the call 

```suggestion
    return this.network.getNftInfo({ canisterId, identity: this.identity, standard });
```